### PR TITLE
Catch up to changing Capistrano 3.5.0

### PR DESCRIPTION
**Pull request by machupicchubeta**
Capistrano 3.5.0 has many changes. 
- https://github.com/capistrano/capistrano/blob/master/CHANGELOG.md#350

In order to notify Slack using Slackistrano, I think that it is necessary to catch up with some changes.

### CODE REVIEW

**Comment by dshmelev** [comment]
:+1:

File: spec/dry_run_spec.rb [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-  before(:each) do
-    Rake::Task['load:defaults'].execute
-  end
-
-  %w[updating reverting updated reverted failed].each do |stage|
+  %w(updating reverting updated reverted failed).each do |stage|
     it "does not post to slack on slack:deploy:#{stage}" do
-      set "slack_run_#{stage}".to_sym, ->{ true }
-      expect(Capistrano::Configuration.env.send(:config)).to receive(:[]).with(:sshkit_backend).and_return(SSHKit::Backend::Printer)
+      set "slack_run_#{stage}".to_sym, -> { true }
```
Comment (dshmelev) [inline spec/dry_run_spec.rb]: Please add this line:
allow(Net::HTTP).to receive(:post_form).and_return double(code: '200' )

Comment (machupicchubeta) [inline spec/dry_run_spec.rb]: @dshmelev Thank you! I got it!

Comment (dshmelev) [inline spec/dry_run_spec.rb]: Sorry, I was a bit wrong... This spec is not correct.
You need to test with dryrun and without. Waiting pls..
I write the correct spec now

Comment (machupicchubeta) [inline spec/dry_run_spec.rb]: Oh....sorry without enough the test.

> I write the correct spec now

Thank you so much! 🍣

Comment (dshmelev) [inline spec/dry_run_spec.rb]: I created PR #47 with our changes. Now waiting for merge =)

Comment (machupicchubeta) [inline spec/dry_run_spec.rb]: Thank you!! I'm looking forward to it! 🙏 

May I close this PR?

Comment (dshmelev) [inline spec/dry_run_spec.rb]: I think yes. =)

Comment (machupicchubeta) [inline spec/dry_run_spec.rb]: Appreciate it, many thanks.

**Pull request closed without merging**